toolchain/check/handle_name.cpp

```diff
@@ -44,38 +44,39 @@ auto HandleMemberAccessExpression(Context& context, Parse::Node parse_node)
   // If the base is a name scope, such as a class or namespace, perform lookup
   // into that scope.
   if (auto name_scope_id = GetAsNameScope(context, base_id)) {
-    auto node_id = name_scope_id->is_valid()
-                       ? context.LookupName(parse_node, name_id, *name_scope_id,
-                                            /*print_diagnostics=*/true)
-                       : SemIR::NodeId::BuiltinError;
-    auto node = context.nodes().Get(node_id);
-    // TODO: Track that this node was named within `base_id`.
-    context.AddNodeAndPush(
+    SemIR::InstId inst_id =
```

Comment:
Preserve `auto` for now:
```suggestion
    auto inst_id =
```